### PR TITLE
DocPseudo

### PR DESCRIPTION
**Pull request by josephaxisa**
DocPseudo

### CODE REVIEW

File: src/scenes/MethodScene/components/DocOperation.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+describe('DocOperation', () => {
+  test.todo('it renders SDK portion of a method')
+
+  test.todo('it renders HTTP portion of a method')
```
Comment (whscullin) [inline src/scenes/MethodScene/components/DocOperation.spec.tsx:4]: TODO for this PR or a different one?

Comment (josephaxisa) [reply src/scenes/MethodScene/components/DocOperation.spec.tsx:4]: jest was failing on this test due to TS errors addressed in another branch. Now that I merged said branch I can give this another shot. I'll circle back.

File: src/components/DocPseudo/DocParam.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { renderWithTheme } from '@looker/components-test-utils'
+import { screen, waitFor, fireEvent } from '@testing-library/react'
+
+import { api } from '../../test-data'
+import { DocParam } from './DocParam'
+
+describe('DocParam', () => {
+  const allParams = api.methods['create_query'].allParams
```
Comment (whscullin) [inline src/components/DocPseudo/DocParam.spec.tsx:9]: I see you changed the eslint config to do this? That shouldn't be necessary. (Referencing object members by string is a de-optimization pattern)

Comment (whscullin) [reply src/components/DocPseudo/DocParam.spec.tsx:9]: Seems like a premature de-optimization.

Comment (jkaster) [reply src/components/DocPseudo/DocParam.spec.tsx:9]: ok. It's not a hill I plan to die on :)

File: src/scenes/MethodScene/components/DocOperation.spec.tsx [hunk 3]
```diff
@@ -7,6 +7,7 @@ import { DocOperation } from './DocOperation'
 
 describe('DocOperation', () => {
   const method = api.methods['create_dashboard']
```
Comment (jkaster) [inline src/scenes/MethodScene/components/DocOperation.spec.tsx:9]:
```suggestion
  const method = api.methods.create_dashboard
```

**Pull request merged**